### PR TITLE
Tourney snakes

### PR DESCRIPTION
**Pull request by joram**

### CODE REVIEW

File: play/apps/tournament/models/tournament_test.py [hunk 1]
```diff
@@ -28,83 +28,86 @@ def _arrange(num_snakes=10):
         snakes.append(snake)
     return snakes, tournament, tournament_brackets
 
-
```
Comment (dlsteuer) [inline play/apps/tournament/models/tournament_test.py]: I'm confused about what's going on here?

File: play/apps/tournament/models/tournaments.py [hunk 2]
```diff
@@ -121,39 +133,44 @@ class Meta:
...
-            if qs.count() == 1 and qs[0] != self:
-                raise SingleSnakePerTeamPerTournamentValidationError()
-
-        models.Model.validate_unique(self, exclude=exclude)
-
-    def save(self, *args, **kwargs):
-        self.validate_unique()
-        super(SnakeTournamentBracket, self).save(*args, **kwargs)
 
+# <<<<<<< HEAD
```
Comment (dlsteuer) [inline play/apps/tournament/models/tournaments.py]: this was a conflict, but now it's just commented out.

File: play/apps/tournament/views/tournament.py [hunk 3]
```diff
@@ -26,28 +27,47 @@ def new(request):
...
-            messages.success(request, f'Tournament "{tournament.name}" successfully updated')
-            return redirect('/tournaments/')
+            t = form.save(commit=False)
+            t.save()
+            ts = TournamentSnake.objects.filter(tournament=tournament)
+            
+            # Remove snakes from the tournament
+            snake_ids = [s.id for s in form.cleaned_data["snakes"]]
+            for remove in ts.exclude(snake_id__in=snake_ids):
+                print('removing this: ', remove)
```
Comment (dlsteuer) [inline play/apps/tournament/views/tournament.py]: do we need this print statement here?

File: play/settings/compose.py [hunk 4]
```diff
@@ -0,0 +1,12 @@
+from settings.base import *
```
Comment (dlsteuer) [inline play/settings/compose.py]: This should be part of this pr, it's unrelated to adding snakes to tournaments.

**Comment by brandonb927** [comment]
I was already working on getting a PR up for this with styling changes fixed here https://github.com/battlesnakeio/play/compare/master...brandonb927:feature-tournament-snakes-improvements

File: play/apps/tournament/models/tournament_test.py (hunk 1 above)
Comment (joram) [reply play/apps/tournament/models/tournament_test.py]: we changed the model that's being used to associate the tournament/bracket/snake, and these tests needed updated. I commented them out and forgot to update them (I'll do that now, sorry for the preemptive pr)

**Comment by brandonb927** [comment]
@dlsteuer John says to close this, so I am. I will take this over.

**Pull request closed without merging**